### PR TITLE
Rc 1.0.25

### LINKED ISSUE #37: Reported latitude seems incorrect
**Describe the bug**

I'm using NMEAReader.parse() to parse a line of text containing a GLL sentence. The resulting longitude is correct, but the latitude is not.

pynmeagps v1.0.24

**To Reproduce**

Steps to reproduce the behaviour:

`msg = NMEAReader.parse('$GNGLL,02348.3822990,S,15313.5862807,E,040856.82,A,D*5F')`
`print(msg)`

output:
`<NMEA(GNGLL, lat=-7.80637165, NS=S, lon=153.2264380117, EW=E, time=04:08:56.820000, status=A, posMode=D)>`

**Expected Behavior**

The output latitude should be -23.80637 degrees.

**Desktop (please complete the following information):**

Windows 10
jupyter_client            8.2.0              pyhd8ed1ab_0    conda-forge
jupyter_core              5.3.0           py311haa95532_0
anaconda Command line client (version 1.11.2)
Python 3.11.3
pynmeagps-1.0.24

**GNSS/GPS Device (please complete the following information as best you can):**

I'm post-processing text captured from a GNSS receiver. Example text shown above.

### PR DESCRIPTION
**Pull request by semuadmin**
# pynmeagps Pull Request Template

## Description

RELEASE CANDIDATE 1.0.25

1. Add support for non-standard 3-digit degree latitude value, e.g.
`'$GNGLL,02348.3822990,S,15313.5862807,E,040856.82,A,D*5F'`
2. NB: footprint of `dmm2ddd` helper method has changed - `att` argument no longer required.

Fixes #37

## Testing

Please test all changes, however trivial, against the supplied pytest suite `tests/test_*.py` e.g. by executing the `tests/testsuite.py` module or using your IDE's native Python unittest integration facilities. Please describe any test cases you have amended or added to this suite to maintain >= 99% code coverage.

- [x] Static tests updated

## Checklist:

- [x] I agree to abide by the code of conduct (see [CODE_OF_CONDUCT.md](https://github.com/semuconsulting/pynmeagps/blob/master/CODE_OF_CONDUCT.md)).
- [x] My code follows the style guidelines of this project (see `CONTRIBUTING.MD`).
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] (*if appropriate*) I have added test cases to the `tests/test_*.py` unittest suite to maintain >= 99% code coverage.
- [x] I have tested my code against the full `tests/test_*.py` unittest suite.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I have [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) my commits.
- [x] I understand and acknowledge that the code will be published under a BSD 3-Clause license.

### CODE REVIEW

File: src/pynmeagps/nmeahelpers.py [hunk 1]
```diff
@@ -124,24 +124,20 @@ def isvalid_cksum(message: object) -> bool:
...
     """
 
     try:
-        if att == LA:
-            posdeg = float(pos[0:2])
-            posmin = float(pos[2:])
-        else:
-            posdeg = float(pos[0:3])
-            posmin = float(pos[3:])
+        dp = pos.find(".")
```
Comment (semudev2) [inline src/pynmeagps/nmeahelpers.py:138]: maybe raise ValueError if find returns -1 (i.e. no dp)?

File: tests/test_static.py [hunk 2]
```diff
@@ -106,11 +106,15 @@ def testBadChecksum(self):
         self.assertEqual(res, False)
 
     def testDMM2DDD(self):
```
Comment (semudev2) [inline tests/test_static.py:108]: as per comment above, maybe add test for missing dp?

**Review by semudev2 — CHANGES_REQUESTED** [review]

File: src/pynmeagps/nmeahelpers.py (hunk 1 above)
Comment (semuadmin) [reply src/pynmeagps/nmeahelpers.py:138]: done

File: tests/test_static.py (hunk 2 above)
Comment (semuadmin) [reply tests/test_static.py:108]: done

**Review by semudev2 — APPROVED** [review]

**Pull request merged**